geonotebook/wrappers/file_reader.py

```diff
@@ -69,6 +71,11 @@ def __del__(self):
         self.dataset.close()
 
     def index(self, *args, **kwargs):
+        target_srs = self.dataset.crs
+        # Annotations are always in WGS84
```

Comment:
Annotations are in wgs84 for now but that may change in the future (distant future)